### DISCUSSION TITLE
Select default view in Preferences (Page, Single Page, Continuous)

### DISCUSSION THREAD

**Discussion by hmturnbull** [musescore/MuseScore#11587] [Ideas]
I noticed in Musescore 3.6 there doesn't seem to be any way to set a default view for new scores.  It would be great if in Preferences you could choose whether new scores start in Page view, Single Page, or Continuous view.  When I'm composing, I almost always work in Continuous view, and it would save time if I didn't have to switch to it every time I start a new composition.

**Comment by somethingstarted** [comment]
what if it remembered what you chose last? there could be an option in settings and or the little thing on the bottom where you choose the view type. then there could be a little text when you hover over it saying "if you click the box, it'll default to this one, but if none of clicked, it'll remember your last choice and just go straight to that one" or something like that.
